Tools/MonoGame.Effect.Compiler/mgfxc_wine_setup.sh

```diff
@@ -15,6 +15,16 @@ then
     exit 1
 fi
 
+# wine 8 minimum needed for dotnet8
+# see https://github.com/MonoGame/MonoGame/issues/8103
+# wine --version will output 'wine-#.# (Ubuntu #.#.#)'
+# grep to find only the 'wine-#' then use sed to remove the 'wine-' prefix
+WINE_VERSION=$(wine --version 2>&1 | grep -oP 'wine-\d+' | sed 's/wine-//')
```

Comment:
Why is grep needed?

Reply:
`wine --version` will output something like

```
wine-#.# (Ubuntu #.#.#)
```

`grep` is used to pattern match the `wine-#` string specifically from thr output, then `sed` to remove the `wine-` prefix which just leaves the major version number of wine detected.

Since it's an integer value and not the #.# floating point value, that makes the version check a simple if statement.

Reply:
`sed` does support regex, but I guess this is fine as well. 

Modify comment to:
```
# wine 8 is the minimum requirement for dotnet 8
# wine --version will output "wine-#.# (Distro #.#.#)" or "wine-#.#"
```
The rest is not needed.

Reply:
@harry-cpp comment modified.